src/shared/components/rentBarChart/rentBarChart.tsx

```diff
@@ -0,0 +1,56 @@
+import { Bar, BarChart, CartesianGrid, Cell, Legend, ResponsiveContainer, Tooltip, XAxis, YAxis } from 'recharts';
+// eslint-disable-next-line import/extensions
+import { isArrayEmpty, randomSetOfColors } from '../../../util/functions';
+
+interface IRentBarChartPropsModel {
+	config: { name: string }[];
+	width?: string;
+	height?: string;
+	backgroundColor?: string;
+	activeLegend: boolean;
+	opacity: number;
```

Comment:
```suggestion
	activeLegend?: boolean;
	opacity?: number;
```
add Optional Chaining (?)

Reply:
Done